x-pack/plugins/reporting/public/management/components/report_diagnostic.tsx

```diff
@@ -50,7 +51,7 @@ const initialState: State = {
   success: true,
 };
 
-export const ReportDiagnostic = ({ apiClient }: Props) => {
+export const ReportDiagnostic = ({ apiClient, configAllowsImages }: Props) => {
```

Comment:
See below for the context, but this seems like the best place unwrap a generic config object and create a boolean called `configAllowsImages `.
```suggestion
export const ReportDiagnostic = ({ apiClient, clientConfig }: Props) => {
  configAllowsImages = clientConfig.export_types.pdf.enabled || clientConfig.export_types.png.enabled;
```